blocks/components/editable/index.js

```diff
@@ -327,51 +334,59 @@ export default class Editable extends wp.element.Component {
 			focus,
 			className,
 			showAlignments = false,
+			inlineToolbar = false,
 			inline,
 			formattingControls
 		} = this.props;
-		const classes = classnames( 'blocks-editable', className );
 
 		// Generating a key that includes `tagName` ensures that if the tag
 		// changes, we unmount (+ destroy) the previous TinyMCE element, then
 		// mount (+ initialize) a new child element in its place.
 		const key = [ 'editor', tagName ].join();
-
-		let element = (
-			<TinyMCE
-				tagName={ tagName }
-				onSetup={ this.onSetup }
-				style={ style }
-				className={ classes }
-				defaultValue={ value }
-				settings={ {
-					forced_root_block: inline ? false : 'p'
-				} }
-				key={ key } />
+		const classes = classnames( className, 'blocks-editable' );
+
+		const formatToolbar = (
+			<FormatToolbar
+				focusPosition={ this.state.focusPosition }
+				formats={ this.state.formats }
+				onChange={ this.changeFormats }
+				enabledControls={ formattingControls }
+			/>
 		);
 
-		if ( focus ) {
-			element = [
-				<Fill name="Formatting.Toolbar" key="fill">
-					{ showAlignments &&
-						<Toolbar
-							controls={ ALIGNMENT_CONTROLS.map( ( control ) => ( {
-								...control,
-								onClick: () => this.toggleAlignment( control.align ),
-								isActive: this.isAlignmentActive( control.align )
-							} ) ) } />
-					}
-					<FormatToolbar
-						focusPosition={ this.state.focusPosition }
-						formats={ this.state.formats }
-						onChange={ this.changeFormats }
-						enabledControls={ formattingControls }
-					/>
-				</Fill>,
-				element
-			];
-		}
+		return (
+			<div className={ classes }>
+				{ focus &&
+					<Fill name="Formatting.Toolbar">
+						{ showAlignments &&
+							<Toolbar
+								controls={ ALIGNMENT_CONTROLS.map( ( control ) => ( {
+									...control,
+									onClick: () => this.toggleAlignment( control.align ),
+									isActive: this.isAlignmentActive( control.align )
+								} ) ) } />
+						}
+						{ ! inlineToolbar && formatToolbar }
+					</Fill>
+				}
 
-		return element;
+				{ focus && inlineToolbar &&
+					<div className="block-editable__inline-toolbar">
+						{ formatToolbar }
+					</div>
+				}
+
+				<TinyMCE
+					tagName={ tagName }
+					onSetup={ this.onSetup }
+					style={ style }
+					className={ classes }
```

Comment:
It's not clear with this component that changing the values later won't affect it. Should it accept classes and styles if that's not the case?